lib/plugins/Label_H1_FPN.test.ts

```diff
@@ -105,17 +110,98 @@ test('decodes Label H1 Preamble FPN in-flight', () => {
+  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.name).toBe('label-h1-fpn');
+  expect(decodeResult.raw.message_timestamp).toBe(1708178417);
+  expect(decodeResult.formatted.description).toBe('Flight Plan');
+  expect(decodeResult.formatted.items.length).toBe(6);
+  expect(decodeResult.formatted.items[0].label).toBe('Route Status');
+  expect(decodeResult.formatted.items[0].value).toBe('Route Planned');
+  expect(decodeResult.formatted.items[1].label).toBe('Origin');
+  expect(decodeResult.formatted.items[1].value).toBe('EHAM');
+  expect(decodeResult.formatted.items[2].label).toBe('Destination');
+  expect(decodeResult.formatted.items[2].value).toBe('KMSP..N55064W000477..N55163W001141..ERAKA..N60000W020000..N61000W030000'); // FIXME - just 'KMSP'
+  expect(decodeResult.formatted.items[3].label).toBe('Arrival Procedure');
+  expect(decodeResult.formatted.items[3].value).toBe('BAINY3');
+  expect(decodeResult.formatted.items[4].label).toBe('Approach Procedure');
+  expect(decodeResult.formatted.items[4].value).toBe('ILS30L(30L)/PR4356,344,360,1060,,,13,,,30,,,P50,M40,36090,,3296,292/DTKMSP,30L,172,215117'); //FIXME - just 'ILS30L'
+  expect(decodeResult.formatted.items[5].label).toBe('Message Checksum');
+  expect(decodeResult.formatted.items[5].value).toBe('0x156d');
+  expect(decodeResult.remaining.text).toBe(':WS:N61000W030000,370..N61000W040000..N60000W050000..URTAK:WS:URTAK,380..LAKES:WS:LAKES,400..N57000W070000..N54300W080000..N49000W090000..DLH..COLDD');
+});
+
+test('decodes Label H1 Preamble FPN with newlines', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_H1_FPN(decoder);
+
+  // https://app.airframes.io/messages/2381936957
+  const text = 'FPN/SN2125/FNQFA780/RI:DA:YPPH:CR:PERMEL001:AA:YMML..MEMUP,S33451E\r\n120525.Y53.WENDY0560'
+   const decodeResult = decoderPlugin.decode({ text: text });
+  console.log(JSON.stringify(decodeResult, null, 2));
+
+  expect(decodeResult.decoded).toBe(true);
+  expect(decodeResult.decoder.decodeLevel).toBe('full');
+  expect(decodeResult.decoder.name).toBe('label-h1-fpn');
+  expect(decodeResult.raw.flight_number).toBe('QFA780');
+  expect(decodeResult.raw.serial_number).toBe('2125');
+  expect(decodeResult.formatted.description).toBe('Flight Plan');
+  expect(decodeResult.formatted.items.length).toBe(5);
+  expect(decodeResult.formatted.items[0].label).toBe('Route Status');
+  expect(decodeResult.formatted.items[0].value).toBe('Route Inactive');
+  expect(decodeResult.formatted.items[1].label).toBe('Origin');
+  expect(decodeResult.formatted.items[1].value).toBe('YPPH');
+  expect(decodeResult.formatted.items[2].label).toBe('Company Route');
+  expect(decodeResult.formatted.items[2].value).toBe('PERMEL001');
+  expect(decodeResult.formatted.items[3].label).toBe('Destination');
+  expect(decodeResult.formatted.items[3].value).toBe('YMML..MEMUP,S33451E120525.Y53.WENDY'); //TODO - pull out route
```

Comment:
Isn't this item the Destination? It would be an ICAO, right?

Reply:
yes, it should be just `YMML` - and then there's the route that's "how to get there"

```>> MEMUP(33.451 S, 120.525 E) > Y53 > WENDY)```

I'm not sure where to put the route right now because if you look above, there can be more route information - i probably should merge but punted on the hard stuff right now